### DISCUSSION TITLE
getting_started/

### DISCUSSION THREAD

**Discussion by giscus** [py32duino/document#2] [comment]
# getting_started/

PY32Duino 是一个由社区推动，旨在适配上海普冉半导体有限公司的 Arm-Cortex 系列 MCU 的 Arduino 框架。 支持芯片 目前我们支持的 MCU 如下： MCU 系列 是否支持 详细链接 :-------: :------: :-----------------: PY32F002A ✅ PY32F003 ✅ PY32F030 ...

https://arduino.py32.org/getting_started/

**Comment by HalfSweet** [comment]
评论测试
